deletecord.user.js

```diff
@@ -140,12 +140,12 @@ async function deleteMessages(authToken, authorId, guildId, channelId, minId, ma
                 const w = (await resp.json()).retry_after;
                 throttledCount++;
                 throttledTotalTime += w;
-                adjustDelay(w); // Adjust delay based on retry_after value
-                log.warn(`Being rate limited by the API for ${w}ms! Increasing search and delete delay...`);
+                //adjustDelay(w); // Adjust delay based on retry_after value
+                log.warn(`Being rate limited by the API for ${w*1000}ms! Consider increasing search delay...`);
                 printDelayStats();
-                log.verb(`Cooling down for ${w * 2}ms before retrying...`);
+                log.verb(`Cooling down for ${w * 3000}ms before retrying...`);
 
-                await wait(w * 2);
+                await wait(w * 3000); //ms to s, 3x delay for margin
```

Comment:
Just updated this to extract 3000. I called it `retryAfterMultiplier`. 

The original code actually waits for 2 times the retryAfter value, which somewhat corroborates the choice of multiplier. The rationale is that you want to pause to create some headroom against the actual rate limit, and this pause should be somewhat proportional to the severity of the rate limit violation. Given that the API's real limit seems to be about 1000ms, waiting around 3s for the typical 1-s retryAfter makes sense to me, but this can be adjusted to taste.